2.0/db_models/pdf_section_embds.py

```
@@ -0,0 +1,40 @@
+    
+    '''
+    # Foreign key to filing_info (the main filing record)
+    accession_number = Column(String(20), 
+                            ForeignKey(f'{settings.FILING_INFO_TABLE}.accession_number'),
+                            nullable=False,
+                            index=True)
+    '''
+    
+    # Foreign key to pdf_sections 
+    section_id = Column(Integer, 
+                       ForeignKey(f'{settings.PDF_SECTIONS_TABLE}.id'),
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)